### ISSUE TITLE
aws-ecs-patterns: Support setting EcsParameters for a scheduled task.

### ISSUE THREAD

**Issue by SydneyUni-Jim** [aws/aws-cdk#25106]
### Describe the feature

Allow other ECS parameters to be set on the Event Bridge rule as part of creating a scheduled EC2 or Fargate task.

The CDK only sets a limited range of ECS parameters on the Event Bridge rule it created to implement the schedule.

* launchType
* platformVersion
* networkConfiguration
* taskCount
* taskDefinitionArn

### Use Case

* Define a `CapacityProviderStrategy` to run scheduled background tasks using  Fargate Spot.
* Set `EnableExecuteCommand` to help in supporting executing scheduled tasks, particularly in triaging borked executions.
* Set `EnableECSManagedTags` to set the `aws:ecs:clusterName` tag for billing.
* Adding custom tags via `TagList`.

### CloudFormation documentation

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-ecsparameters.html

### Call path

This is the call path used to establish the additional parameters on the EventBridge rule target.

1. `ScheduledFargateTask.constructor`
2. `ScheduledTaskBase.addTaskAsTarget`
3. `events.Rule.addTarget`
4. `eventTargets.ECSTask.bind`

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.73.0

### Environment details (OS name and version, etc.)

macos/13.3 (aarch64) nodejs/18.15.0 typescript/5.0.4

**Comment by pahud** [comment]
Yes this makes perfect sense to me. I am making it p2 and welcome any PRs from the community.

**Comment by blimmer** [comment]
This PR looks to be relevant to this: https://github.com/aws/aws-cdk/pull/23838

**Comment by rdbatch02** [comment]
> This PR looks to be relevant to this: #23838

Hey @blimmer perfect timing on this, I just opened a PR (#25222  to ecs-patterns to hoist that tagging functionality into this module from my last PR.
